src/popup/pages/LanguageSettings.vue

```diff
@@ -4,44 +4,40 @@
       {{ $t('pages.languageSettings.chooseLanguage') }}
     </p>
 
-    <div class="languages">
-      <RadioButton
-        v-for="{ code, name } in list"
-        :key="code"
-        :value="active && active.name == name"
-        :disabled="false"
-        :class="['language', {active: active && active.name == name}]"
-        @input="switchLanguage(code)"
-      >
-        <div
-          class="row"
-          @click="switchLanguage(code)"
-        >
-          <div>
-            {{ name }}
-            <span>({{ code }})</span>
-          </div>
-        </div>
-      </RadioButton>
-    </div>
+    <RadioButton
```

Comment:
I don't see a reason for removing the DIV that wraps the options.
In my opinion it's good to wrap lists with div, even if the div does nothing in terms of styling.